### DISCUSSION TITLE
Koha-Suomen hallituksen tiedote 5/2023 julkaistu

### DISCUSSION THREAD

**Discussion by AnneliO** [KohaSuomi/Koha#546] [Tiedotteet]
[Koha-Suomen hallituksen tiedote 5/2023](https://koha-suomi.fi/hallituksentiedotteet2023#koha-suomen-hallituksen-tiedote-52023) on nyt julkaistu Koha-Suomen verkkosivuilla.

**Comment by AnttHeik** [comment]
Mistäköhän se johtuu, että nuo linkkeinä olevat pdf-tiedostot eivät avaudu näissä? GitHubissa tulee vain virheilmoitus: Unable to render code block

**Reply by AnneliO** [reply]
Jaha, miepä koitan laittaa ne eri tavalla saataville.

**Reply by AnneliO** [reply]
Mites nyt Antti?

**Reply by AnneliO** [reply]
Päivitä sivu ennen kokeilua. :)

**Reply by AnttHeik** [reply]
Joo, nyt aukeavat hyvin :)
